src/modules/recipe/compositionDynamic/compositionDynamic.html

```diff
@@ -0,0 +1,16 @@
+<template>
+    <ui-card title="CompositionDynamic">
+        <ui-button label="Load Hello Component" onclick={loadComponent}>
+        </ui-button>
+
+        <div class="dynamic-component">
+            <recipe-dynamic-import
```

Comment:
We may want to add a comment about the `recipe-dynamic-import` tag and the `lwc:dynamic` attribute. Users may try to look for a `recipe-dynamic-import` component in our project...

Reply:
Done